src/modules/missionHelper/i18n/en_US.json

```diff
@@ -43,12 +43,12 @@
 			"arff": "ARFF",
 			"battalion_chief_vehicles": "Battalion Chief Vehicle",
 			"boats": "Boat",
-			"brush_truck": "Wildland fire engines",
+			"brush_truck": "Wildland Fire Engines",
 			"fbi": "FBI Units",
 			"fbi_bomb": "FBI Bomb Technician Vehicle",
 			"fbi_drone": "FBI Surveillance Drone",
-			"fbi_investigation": "FBI Investiogation Wagon",
-			"fbi_mcc": "FBI Mobile Command Centers Vehicle",
+			"fbi_investigation": "FBI Investigation Wagon",
+			"fbi_mcc": "FBI Mobile Command Center Vehicle",
```

Comment:
We could also use singular + plural over here.
Please check all the requirement values wether we could use singular and plural ^^

Reply:
b2164e1